src/main/java/frc/robot/constants/swerve/DriveConstants.java

```diff
@@ -115,16 +115,23 @@ public class DriveConstants {
   public static double kHeadingP= 1;//4
   public static double kHeadingD= 0;
 
-   //balance PID
-   public static double kBalanceP= 0.01;
-   public static double kBalanceI= 0;
-   public static double kBalanceD= 0;
-   public static final double kBalanceMaxOutput = 0.5;
-   public static final double kBalanceTolerance = 0.1;
+  //balance PID
+  public static double kBalanceP= 0.05;
+  public static double kBalanceI= 0;
+  public static double kBalanceD= 0;
+  public static final double kBalanceMaxOutput = 0.5;
```

Comment:
Comment on what MaxOutput and Tolerance are?

Reply:
I removed tolerance as it wasn't used, theoretically it would be used for checking if the command is finished, but I would like it to run until the end of auto. For teleop, i want it to end when the driver releases the button.

Reply:
max output comment also added